src/headless/package.json

```diff
@@ -35,11 +28,13 @@
     "url": "https://github.com/conversejs/converse.js/issues"
   },
   "gitHead": "9641dcdc820e029b05930479c242d2b707bbe8e2",
-  "devDependencies": {},
   "dependencies": {
+    "@converse/openpromise": "^0.0.1",
     "@converse/skeletor": "conversejs/skeletor#c7e76b023337a8ac201c74507f7a1209de1c6d18",
     "dayjs": "1.10.6",
+    "dompurify": "^2.3.1",
     "filesize": "^7.0.0",
+    "lit": "^2.0.0-rc.2",
```

Comment:
I don't think we should add `lit` as a dependency since it's a UI framework and `@converse/headless` is supposed to be UI agnostic.

I see that we're only using `lit-tml` and only in one file (src/headless/plugins//emojis/index.js), so at most we should add `lit-html` as dependency and then import from that, but I think what would be best would be to refactor the emojis plugin so that it doesn't require `lit-html` in `@converse/headless` and instead move code that relies on `lit-html` to the UI part of Converse.

Reply:
I've now refactored @converse/headless to remove the dependency on `lit-html`

https://github.com/conversejs/converse.js/commit/9d4382c7540b8fc635652ba32a4a7b7ed6671cfb